ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ExpertRowMapper.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire.CCDExpertReport;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.ExpertReport;
+
+@Component
```

Comment:
ExpertReportRowMapper pls. It needs to be obvious which mapping it does. Its not mapping Expert.